modules/environment/login/proot-attrs/aarch64.nix

```diff
@@ -0,0 +1,5 @@
+# WARNING: This file is autogenerated by the deploy script. Any changes will be overridden
+{
+  url = "file:///data/local/tmp/n-o-d/bootstrap-aarch64.zip";
```

Comment:
I presume, the committed version will have my bootstrap url here instead?

Isn't there a dependency loop then, with the bootstrap tarball containing the hash of itself?

Reply:
"will have my bootstrap url here instead" yes

"isn't there a dependency loop"
No, two things:
- the hash is only a hash of the proot binary
- during bootstrap building prootStatic is overridden, so this is never evaluated